packages/node/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.0.2] - 2023-10-04
+### Fixed
+- Fix reindex service without poi feature with `node-core`
+
+### Changed
+- Version bump with `types-core` 0.1.1
```

Comment:
I don't think this applies here

Reply:
I think suppose be `types-ethereum` and `common-ethereum`